.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
```

Comment:
Do we want to add these?

```yaml
  repository_dispatch:
  workflow_dispatch:
```

Maybe not necessary to trigger it manually?

Reply:
This is a long process. It will trigger the whole code On schedule. I recommend starting using it and see what we get from the current code.

Reply:
WFM.